products/opensuse/main.pm

```diff
@@ -411,6 +411,7 @@ else {
         || is_systemd_test())
     {
         loadtest "console/system_prepare";
+        loadtest "x11/disable_screensaver" if (check_var('FLAVOR', 'NET') && check_var('UPGRADE', '1') && heck_var('ORIGINAL_VERSION', '15.0'));
```

Comment:
Typo

```suggestion
        loadtest "x11/disable_screensaver" if (check_var('FLAVOR', 'NET') && check_var('UPGRADE', '1') && check_var('ORIGINAL_VERSION', '15.0'));
```

Reply:
I can find leap 15.1 upgrade test has the same problem. https://openqa.opensuse.org/tests/4601162, let me run VR as well

Reply:
New VR passed as well